Article/Article.js

```diff
@@ -26,4 +39,5 @@ class Article {
 
 */
 
-let articles;
\ No newline at end of file
+let articles = document.querySelectorAll('.article');
+articles.forEach( article => new Article(article));
```

Comment:
Looks great here - if I may suggest, you can also condense this to a single line by adding the forEach method directly to the end of line 42